packages/slate-editor/src/extensions/web-bookmark/WebBookmarkExtension.tsx

```diff
@@ -24,6 +25,19 @@ export const WebBookmarkExtension = ({
             [BOOKMARK_NODE_TYPE]: createDeserializeElement(parseSerializedElement),
         }),
     },
+    isElementEqual: (node, another) => {
+        if (isBookmarkNode(node) && isBookmarkNode(another)) {
+            // Compare ignoring `uuid` and `children`
+            return (
+                node.url === another.url &&
```

Comment:
Am I right that we should not forget to update `isElementEqual` once we will add some new attributes to a node?

Reply:
Yes, you are right.

For this reason, I've been thinking about using `isEqual(omit())` instead of explicitly comparing every prop, but in the end decided that runtime performance gain (even a small one) is worth pursuing.

Reply:
Modification detection will start to produce false-negative results, marking the content as not modified, even though some property is different.